doc/architecture/consensus/index.rst

```diff
@@ -8,7 +8,12 @@ Below, we give an overview over the nodes state machine in both settings and the
 CFT Consensus Protocol
 ----------------------
 
-The crash fault tolerant implementation in CCF is based on Raft. You can find more information on the Raft implementation of reconfiguration in CCF :doc:`here <1tx-reconfig>`.
+The crash fault tolerant implementation in CCF is based on `Raft <https://raft.github.io/>`_. The key differences between the original Raft protocol (as described in the `Raft paper <https://raft.github.io/raft.pdf>`_), and CCF Raft are as follows:
+
+* Transactions in CCF Raft are not considered to be committed until a subsequent signed transaction has been committed. More information can be found :doc:`here </consensus/merkle_tree>`. Transactions in the ledger before the last signed transactions are discarded during leader election.
+* As well as Raft's two-phase reconfiguration protocol, CCF also supports one-phase reconfiguration. You can find more information on the Raft implementation of reconfiguration in CCF :doc:`here <1tx-reconfig>`. Note that CCF Raft does not support node restart. If a node fails and starts again, it must rejoin Raft via reconfiguration.
```

Comment:
```suggestion
* As well as Raft's two-phase reconfiguration protocol, CCF also supports one-phase reconfiguration. You can find more information on the Raft implementation of reconfiguration in CCF :doc:`here <1tx-reconfig>`. Note that CCF Raft does not support node restart as the unique identity of each node is tied to the node process launch. If a node fails and is replaced, it must rejoin Raft via reconfiguration.
```

I think would be slightly clearer. 

Also, the wording of this sentence suggests that the two-phase reconfiguration protocol is the default reconfiguration protocol, but it's actually the opposite (1tx is default and currently the one we recommend for production deployments) so it may be worth switching the order around.